lib/components/artist/artworks/grid.js

```diff
@@ -118,7 +118,7 @@ class ArtworksGrid extends React.Component {
       // There are artworks without images and other ‘issues’. Like Force we’re just going to reject those for now.
       // See: https://github.com/artsy/eigen/issues/1667
       //
-      if (artwork.image) {
+      if (artwork.image && artwork.partner) {
```

Comment:
No let's show the artwork but remove the label, as per your earlier commit. We'll deal with the artwork separately later. It's an edge case anyways.